src/AvInput.js

```diff
@@ -30,14 +30,23 @@ export default class AvInput extends AvBaseInput {
       valueParser: omit7,
       valueFormatter: omit8,
       className,
-      tag: Tag,
+      tag,
       getRef,
       id = this.props.name,
       ...attributes
     } = this.props;
 
     const touched = this.context.FormCtrl.isTouched(this.props.name);
     const hasError = this.context.FormCtrl.hasError(this.props.name);
+    let Tag = tag;
+
```

Comment:
you need to add `let tags;` here otherwise the destructuring below will throw an undefined variable error.